### ISSUE TITLE
[kn-admin] kn-admin has different behaviour on minikube knative

### ISSUE THREAD

**Issue by chaozbj** [knative/client-contrib#42]
#### Environment
System: MacOS 10.15.4
Minikube: v1.11.0
Kubectl client: v1.15.5
Kubectl server: v1.18.2
Knative: v0.12.0

#### Commands to Run
1. ```./kn-admin domain set -d test.com --selector app=prod --selector app=stage```
2. ```./kn-admin autoscaling update --scale-to-zero```

#### Results and Logs
1. For the domain command, I checked the ```config-domain```, the **test.com** with the selectors is set, but there is an unexpected line ```----``` between domain and selector:
```yaml
test.com:
----
selector:
  app: prod
  app: stage

Events:  <none>
```
2. For the autoscaling command, I found that the ```enable-scale-to-zero``` is set to with boolean value instead of string value:
```yaml
enable-scale-to-zero:
----
true
Events:  <none>
```

I talked to @zhanggbj and she said there is no redundant line: ```----``` and the value is set to string not boolean on kube cluster. We will take time to investigate why the behaviour is different on minikube.

**Comment by zhanggbj** [comment]
@chaozbj good findings!
I did tests on IBM IKS clusters and it works fine, there's some error for minikube support. Need further investigation. Thx!

**Comment by chaozbj** [comment]
I wrote a simple application to change a configMap in ```default``` namespace, I planed to open an issue in minikube repo to ask and use this app as the example to explain my question. But when I tried to run it locally with my minikube, I can't reproduce this issue now. And then, I tried to use kn-admin to set a domain, still can't reproduce. 
I don't know why, the only difference is that I recreated minikube cluster and reinstalled knative, but their versions are same.
Since I can't reproduce it, I would like to close it.
